play-framework/README.md

```diff
@@ -6,3 +6,4 @@ This module contains articles about the Play Framework.
 - [REST API with Play Framework in Java](https://www.baeldung.com/rest-api-with-play)
 - [Routing In Play Applications in Java](https://www.baeldung.com/routing-in-play)
 - [Introduction To Play In Java](https://www.baeldung.com/java-intro-to-the-play-framework)
+- [Asynchronous HTTP Programming with Play Framework](https://www.baeldung.com/async-http-with-play)
```

Comment:
We don't add this link at this stage. Let's remove it.

Reply:
ok. removed.